src/core/constants.js

```diff
@@ -48,6 +49,8 @@ export const ADDON_TYPE_LANG = 'language';
 export const ADDON_TYPE_OPENSEARCH = 'search';
 export const ADDON_TYPE_STATIC_THEME = 'statictheme';
 export const ADDON_TYPE_THEME = 'persona';
+export const ADDON_TYPE_THEMES_FILTER = config.get('enableStaticThemes') ?
+  `${ADDON_TYPE_THEME},${ADDON_TYPE_STATIC_THEME}` : ADDON_TYPE_THEME;
```

Comment:
should I be testing this config setting?

Reply:
I think so yes, and that's why I don't think we should be doing this.

Could we have a `getAddonTypeFilter()` helper function instead? I think this method could take the `addon` and `_config = config` as arguments, and return the filter for the API call (which is located in the sagas, right?)

I would also try to avoid replacing `addon.type` to `addon.addonFilter` as much as I could (ideally we should not add `addonFilter` to the addon type, but I might miss something).

Reply:
@willdurand, thanks for quick feedback. Let me revisit this..